modules/permissions/common_permissions.tf

```diff
@@ -120,8 +128,10 @@ resource "aws_iam_policy" "custodian_get_parameters" {
 }
 
 resource "aws_iam_role" "jenkins_lambda_deploy_role" {
```

Comment:
This role can be deleted as it relates to the old Jenkins CI/CD pipeline which has been removed.